eng/pipelines/ci-template.yml

```diff
@@ -82,14 +82,17 @@ steps:
     - script: pnpm run lint --command pylint
       displayName: Pylint
       workingDirectory: $(Build.SourcesDirectory)/autorest.python/packages/${{parameters.folderName}}
+      condition: and(succeeded(), eq('${{parameters.folderName}}', 'autorest.python'))
```

Comment:
There is no additional src code of .py file in `typespec-python` so no need to check lint issues.